conda_build/jinja_context.py

```diff
@@ -391,11 +391,25 @@ def compiler(language, config, permit_undefined_jinja=False):
 
         # support cross compilers.  A cross-compiler package will have a name such as
         #    gcc_target
-        #    gcc_linux-cos5-64
+        #    gcc_linux-cos6-64
         compiler = '_'.join([compiler, config.variant['target_platform']])
     return compiler
 
 
+def cdt(package_name, config, permit_undefined_jinja=False):
```

Comment:
I can't add tests to cdt until we've locked down what a cdt is. This is enough to get me past the speed bump of needing to install cdt packages.